fs.js

```diff
@@ -285,4 +336,11 @@ function getFs(path) {
 	return new FSPath(path);
 }
 
-module.exports = getFs;
+const FS = Object.assign(getFs, {
+	/**
+	 * @type {Map<string, [Promise, (() => string | Buffer)?, Object]>}
+	 */
+	pendingUpdates: new Map(),
```

Comment:
Is it important to be able to access this from outside the module? I would've just made it a constant and handled the logic for it from a `pendingUpdate` getter/setter on `FSPath`. I'm a bit wary of needing to mutate `getFs`, but I'll admit it doesn't make any real difference whether it is or not.

Reply:
PS convention is to leave all variables public and globally accessible, for ease of manual hotpatching. We couldn't possibly hotpatch `writeUpdate` if it referred to a private variable, etc.

As for mutating `getFs`, literally all our libraries work this way. `Users`, `Rooms`, `Ladders` are all functions with properties. It's a bit weird, but it makes for a very convenient API.